### ISSUE TITLE
The code implementation of ParameterNet is not completely consistent with the paper description

### ISSUE THREAD

**Issue by g1239** [huawei-noah/Efficient-AI-Backbones#242]
![image](https://github.com/huawei-noah/Efficient-AI-Backbones/assets/54445180/922a5e8d-1222-4f6a-940b-00eba9549391)
Thanks very much for your excellent job, but I found that the routing module of dynamic convolution in the code is implemented by using CondConv.2d and sigmoid rather than Dynamic-conv and softmax. To my knowledge, this will lead to a significant difference in the training dynamics and the final convergence accuracy.  And I wonder if there is any potential reason for this?

**Comment by iamhankai** [comment]
Dynamic conv is a general concept. It can be implemented in several manners. We used the implementation in https://arxiv.org/pdf/1904.04971.pdf

**Comment by g1239** [comment]
Thank you for your timely and helpful respons. But does this mean that the description of the model in the paper-v2 published on arxiv is still not the latest version? Looking forward to seeing the official version published in cvpr2024 as soon.
